app/views/users/new.html.erb

```diff
@@ -1,26 +1,26 @@
 <div class="col-md-3">
 
-<h3>What is Public Lab?</h3>
```

Comment:
Hmm, this is a lot of individual lines -- is there a way to save whole HTML blocks, translated? That would make maintaining this, or changing it, much easier.